Vas 47/feat/register

aayush-musyaju opened the pull request:
Implements the user registration page!

aayush-musyaju committed:
Adds register form

New user can register into the application using the register form.

aayush-musyaju committed:
updates the cards width and height of the register page and login page

aman007shrestha commented on src/components/registerForm/RegisterForm.tsx:
```diff
@@ -0,0 +1,92 @@
+   * @param {registerInterface} values
+   */
+  const onFinish = (values: registerInterface) => {
+    console.log(values);
```
I assume password comparison with confirm password is yet to do.

aman007shrestha reviewed (approved):
Looks good to me.

aayush-musyaju committed:
Checks whether the password and confirm password are same

The error message is shown if the password and confirm password doesn't
match else navigates to the login page.
